tools/bouncer_worker/src/lib/utils.js

```diff
@@ -26,4 +28,6 @@ Utils.sleep = (ms) => new Promise((resolve) => {
 	setTimeout(resolve, ms);
 });
 
+Utils.getCurrentDateTimeAsString = () => moment().format('DD-MM-YY_HH[h]mm[m]ss[s]');
```

Comment:
Can we format this YYYY-MM-DD_HH[h]mm[m]ss[s] please to be an iso recognised date format.